docs/source/install-tutorial.rst

```diff
@@ -3,19 +3,32 @@
 Installation tutorial with Debian, NGinx and gunicorn
 =====================================================
 
-preliminary packages:
+Following this tutorial you'll have a running Debian+Nginx+gunicorn+bepasty (HTTP) instance using Python3 on a minimally installed Debian server.  It is strongly recommended to use HTTPS for Internet facing services - configuring Nginx for HTTPS is left to the reader.
+
+After installing a minimal Debian 10 server.
+
+Install preliminary required packages:
 
 ::
 
-  apt-get install build-essential nginx supervisor python-dev git-core python-pip python-virtualenv
+  apt-get install nginx git-core python3-dev python3-pip virtualenv python3-virtualenv
```

Comment:
so, you did not need build-essential? is it installed on your base system already?

what's up with the supervisor stuff?

Reply:
Good spot regarding the missing supervisor.

On Debian 10 - no need for build-essential as anything that is required by the python/pip/etc.. packages are pulled in anyway.

I have now installed twice more to verify and I have confirmed that all you need is:
apt-get install git-core python3-dev python3-pip virtualenv python3-virtualenv nginx supervisor